src/GeneralStateTestsFiller/stTransactionTest/Opcodes_TransactionInitFiller.json

```diff
@@ -323,7 +369,9 @@
                 ":raw 0x6000600060006000730f572e5295c57f15886f9b263e2f6d2d6c7b5ec6612710fa5060006000f3", 
                 ":raw 0x60006000fd60006000f3", 
                 ":raw 0x32ff",
-                ":label invalid_first_byte_ef :raw 0xef"
+                ":label invalid_first_byte_ef :raw 0xef",
+                ":label side_effects :yul { pop(call(50000, 0xb94f5374fce5edbc8e2a8697c15331677e6ebf0b, 0, 0, 0, 0, 0)) }"
```

Comment:
This was missing a comma in the end of the line, how did it work and didn't complain? @winsvega 

Did it consider this a concatenation of two strings?

Reply:
yes apparently I can parse without `,`

Reply:
this PR however fails on master geth. which version did you fill it with?

Reply:
> this PR however fails on master geth. which version did you fill it with?

It was this commit according to generated test
https://github.com/ethereum/go-ethereum/commit/addd8824cf3ad6133c1b1bbc3387a621eafba6a3

Reply:
The current `develop` version of this test passes for me with latest geth https://github.com/ethereum/go-ethereum/commit/017cf71fbd51ede510dd0e40e02f89b5340eea93